maxim/tests/test_add_dataset_entries.py

```
@@ -0,0 +1,348 @@
+            first_entry = parsed_body["entries"][0]
+            self.assertEqual(first_entry["input"]["payload"], {"text": "object input"})
+            self.assertEqual(first_entry["expectedOutput"]["payload"], {"text": "object output"})
+            
+            # Verify second entry (dict)
+            second_entry = parsed_body["entries"][1]
+            self.assertEqual(second_entry["input"]["payload"], "dict input")
+            self.assertEqual(second_entry["expectedOutput"]["payload"], "dict output")
+
+    def test_add_dataset_entries_invalid_entry_type(self) -> None:
+        """Test that add_dataset_entries raises Exception for invalid entry types."""
+        with self.assertRaises(Exception) as context:
```

Ruff (0.12.2)

Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)